### DISCUSSION TITLE
Extended moon names - can we switch it off?

### DISCUSSION THREAD

**Discussion by ghrasko** [Stellarium/stellarium#3745] [General]
For some time Solar system moon names are extended containing the scientific designation, i.e.: "(SV) Rhea". For most of the use cases this seems to be unnecessary. Can it be switched off somewhere, so that it is "Rhea" only? I think, this should be even the default.

Any information or ideas?

Gábor

**Comment by l-gym** [comment]
I also have the same problem, and it's something new. For custom users there's no need for the moons' designation (this designation number is not even included in the official IAU name).  Thanks for the possible solution!
Monika

**Reply by alex-w** [reply]
The moon numbers is official IAU designation...

**Reply by l-gym** [reply]
Then why were they user friendly earlier? :) 
The name in Stellarium (I have ver. 24.1 now, but it turned this way in 23.4 version) is e.g. "(SV) Rhea" and the official designation is "Saturn V (Rhea)"
I'm sure nobody needs the long version. It makes hard to see where the moons themselves are.
Thanks again!!!
![stellarium-005](https://github.com/Stellarium/stellarium/assets/170166017/9cc3a41f-b0dd-4192-bf39-2e3cd2d346f6)

**Reply by 10110111** [reply]
Wouldn't it be better to first show the common name, and only then the number?

**Reply by alex-w** [reply]
Please check the fresh version (development snapshot) of Stellarium:
https://github.com/Stellarium/stellarium-data/releases/tag/weekly-snapshot

**Reply by l-gym** [reply]
Looks like you're a kind and helpful person. <3 Thanks a lot!

**Comment by alex-w** [comment]
> For some time Solar system moon names are extended containing the scientific designation, i.e.: "(SV) Rhea". For most of the use cases this seems to be unnecessary. Can it be switched off somewhere, so that it is "Rhea" only? I think, this should be even the default.

Short answer: no, it's not possible

Long answer: IAU numbers for moons (its scientific designations) were added to resolve problem of finding asteroids and moons with same names in one hand and in other hand adding consistence for naming celestial bodies. Today I've removed displaying the IAU moon numbers from screen labels (of course it will touches Search Tool, AstroCalc Tool and Scripting Engine for finding moons by their names)

**Comment by alex-w** [comment]
Please check the fresh version (development snapshot) of Stellarium:
https://github.com/Stellarium/stellarium-data/releases/tag/weekly-snapshot

**Reply by ghrasko** [reply]
Thanks very much!
